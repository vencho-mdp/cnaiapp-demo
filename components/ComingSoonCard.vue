<template>
  <div
    class="bg-gray shadow-md h-20 w-80 sm:w-96 p-4 rounded-md flex items-center justify-center"
  >
    <v-subtitle class="!text-white"><slot /></v-subtitle>
  </div>
</template>

<script>
export default {};
</script>

<style></style>
