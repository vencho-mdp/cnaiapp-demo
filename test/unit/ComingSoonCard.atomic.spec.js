import { mount } from '@vue/test-utils'
import ComingSoonCard from '@/components/ComingSoonCard.vue'

describe('ComingSoonCard', () => {
  it('should render correctly', async () => {
    const el = mount(ComingSoonCard, {
      slots: {
        default: '<span> Proximamente </span>'
      }
    })
    const text = el.find('h2 > span').text()
    const html = el.html()
    expect(text).toBe('Proximamente')
    expect(html).toMatchSnapshot()
  })
})
