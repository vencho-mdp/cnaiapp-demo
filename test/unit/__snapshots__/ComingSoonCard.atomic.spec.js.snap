// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComingSoonCard should render correctly 1`] = `
"<div class=\\"bg-gray shadow-md h-20 w-80 sm:w-96 p-4 rounded-md flex items-center justify-center\\">
  <h2 class=\\"text-base font-bold !text-white\\"><span> Proximamente </span></h2>
</div>"
`;
